terraform/providers.tf

```diff
@@ -3,37 +3,77 @@
 # used to assume the roles required to access remote state in the
 # Terraform backend.
 provider "aws" {
+  default_tags {
+    tags = var.tags
+  }
   region = var.aws_region
 }
 
-# The provider used to create roles that can read objects from the
-# production COOL "third-party" bucket
+# The provider used to create the role that can be assumed to do
+# everything the CI user needs to do in the staging account.
 provider "aws" {
-  alias  = "images-production"
+  alias = "images_staging_provisionaccount"
```

Comment:
Can you please re-order these providers alphabetically so that `aws.images_production_provisionaccount` comes before `aws.images_staging_provisionaccount`?

Reply:
Please see commit 99bfd75.